### PR TITLE
Getting Polyphemus to run on gorgonzola

### PR DESCRIPTION
**Pull request by sa2257**
This PR 
- configures toolchain and hls command prefix
- includes `make`, `g++`, `curl` and `bash` for running Xilinx tools from the containers

### CODE REVIEW

**Comment by sa2257** [comment]
Need to iron out a couple of weird issues,
- docker containers throw `invalid mount config for type "volume": invalid mount path:` 
might be due to workproc created with name `f3f593bfd8c9_seashell_workproc_1`
- `seashell-instance/buildbot.cfg` configurations don't seem to take affect

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,8 @@ MAINTAINER Adrian Sampson <asampson@cs.cornell.edu>
 MAINTAINER Rachit Nigam <rnigam@cs.cornell.edu>
 
 # Add OpenSSH and the sshpass utility for Zynq execution.
-RUN apk add openssh sshpass
+RUN apk update && apk add bash curl
+RUN apk add make g++ openssh sshpass
```
Comment (sampsyo) [inline Dockerfile]: Just to clarify, can you please explain briefly why `make` and `g++` are required in the Polyphemus container? I thought they'd only be required in our Xilinx container, where the actual builds take place.

File: polyphemus/config_default.py [hunk 2]
```diff
@@ -5,7 +5,7 @@
 # example, if your deployment needs to execute the Xilinx tools on a different
 # machine or a different Docker container, a deployment could use this prefix
 # to send these commands there.
-HLS_COMMAND_PREFIX = []
+HLS_COMMAND_PREFIX = ['/polyphemus/instance/onxilinx.sh']
```
Comment (sampsyo) [inline polyphemus/config_default.py]: These Gorgonzola-specific config options need to go in our private config, not in the default config values (as you noted above).

Comment (sa2257) [reply polyphemus/config_default.py]: Yes, the private config doesn't seem to take effect though.

File: Dockerfile (hunk 1 above)
Comment (sa2257) [reply Dockerfile]: Yes, but buildbot fails at https://github.com/cucapra/polyphemus/blob/96f996c1db847dc128974a03ab09fea21c37d7c6/polyphemus/worker.py#L261 on workproc container when it calls the Xilinx container to do the build without them. 

I added bash to test from within the container.

**Comment by sampsyo** [comment]
I tracked down the problem where the configuration was being ignored: the file needed to be renamed to `polyphemus.cfg`. 🤦‍♂ Fixed in our infrastructure in https://github.com/cucapra/seashell-gorgonzola/commit/0c713d8aa64a2a40b98ea764264ef88bea3be6bd.

File: Dockerfile (hunk 1 above)
Comment (sampsyo) [reply Dockerfile]: But that executes `make` in the Xilinx container, not in the web server container. (Hence the prefix.)

**Pull request closed without merging**